torch/csrc/jit/script/compiler.cpp

```diff
@@ -890,6 +890,18 @@ struct to_ir {
     graph->insertNode(continue_node);
   }
 
+  void emitDelete(const Delete& stmt) {
+    Subscript subscript(stmt.expr());
```

Comment:
Expecting a subscript expr without checking is going to give an incomprehensible error if someone tries to use `del` on a variable, which is valid python.
```
a = 5
del a
```
This also doesn't handle other valid `del` cases:
```
li = [1, 2, 3]
del li[1:3]
print(a) # [1]
```